cloudflare.services.yml

```diff
@@ -11,3 +11,8 @@ services:
     arguments: ['%cloudflare.cache_tag_header_limit%']
     tags:
       - { name: event_subscriber }
+  cloudflare:
```

Comment:
I buy into the namespacing in the name.  Is .service any better than no namespace?  If this is the paradigm we are buying into it probably needs to be a little more descriptive.  cloudflare.basicsettings and cloudflare.invalidator